adr/00002-theming-by-users.md

```diff
@@ -0,0 +1,22 @@
+# 00001 - Theming by users
+
+# Status
+
+Decided
+
+# Context
+
+MacawUI exposes the design tokens that allows to change semantic elements of the UI by using css variables. However it's possible to change value in these variables, it can caused issues on UI compatibility, thus it must be somehow limited.
```

Comment:
> it can caused issues on UI compatibility

That depends on how theming is approached if the theme author uses color tokens (instead of singletons) then the theme will be coherent

Reply:
Correct, the thing is you should change theme values only within the root (look at notes), not eg. from single app level as it will affect only that app. In the future we will try to expose perhaps some theming API so the user have (controlled) way to it. This decision is mainly about that.